From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Wed, 30 Nov 2022 09:54:32 +0100
Subject: application: Switch links to gtk-3-24 branch

`master` doesn't exist anymore and we want to refer to a GTK3 example

Origin: upstream, 3.24.36, commit:fb72264a2781de5af7021e9292464746d4a8bab8
---
 gtk/gtkapplication.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gtk/gtkapplication.c b/gtk/gtkapplication.c
index 205d377..17e9205 100644
--- a/gtk/gtkapplication.c
+++ b/gtk/gtkapplication.c
@@ -113,7 +113,7 @@
  *
  * ## A simple application ## {#gtkapplication}
  *
- * [A simple example](https://git.gnome.org/browse/gtk+/tree/examples/bp/bloatpad.c)
+ * [A simple example](https://gitlab.gnome.org/GNOME/gtk/-/blob/gtk-3-24/examples/bp/bloatpad.c)
  *
  * GtkApplication optionally registers with a session manager
  * of the users session (if you set the #GtkApplication:register-session
